docker-compose.yml

```diff
@@ -45,16 +45,13 @@ services:
     depends_on:
       - influxdb
     command: sh -c 'sleep 5; influx setup
-     --bucket "${INFLUX_TLM_BUCKET:-delfi_pq_downlink}"
+     --bucket "${INFLUX_BUCKET:-default}"
      --token "${INFLUXDB_V2_TOKEN:-adminpwd}"
      --org "${INFLUXDB_V2_ORG:-Delfi Space}"
      --username=${INFLUX_USERNAME:-"admin"}
      --password=${INFLUX_PASSWORD:-"adminpwd"}
      --host=http://influxdb:8086
-     --force &&
-     influx bucket create --name "${INFLUX_DATA_BUCKET:-delfi_pq_raw_data}"
-     --token "${INFLUXDB_V2_TOKEN:-adminpwd}"
-     --org "${INFLUXDB_V2_ORG:-Delfi Space}"'
+     --force'
```

Comment:
The credentials for influxdb are set here. To log into the admin site via the GUI at `localhost:8086`, you can use the username and password `admin: adminpwd`. While with the imfluxdb client, the organization and token are used `Delfi Space: adminpwd`. I don't see `adddelfipqdata` getting stuck in a loop, but it does take about 15 min to run on my end.

Reply:
Great! The problem seems fixed... it takes quite some time but it successfully loads stuff in InfluxDB. Data is also persistent so we are really moving forward.

I also noticed that I can keep using the server in the meantime (I mean browsing the web server): is the scheduler running in a separate thread or does it simply contain more "sleep" calls to let the server handle web traffic?

Reply:
Nice! Shall we merge this one then? The scheduler runs in a separate thread in the background.

Reply:
Yes!